public/locales/fr/translation.json

```diff
@@ -2,60 +2,63 @@
     "sign_in": "Se connecter",
     "sign_out": "Se déconnecter",
     "sign_up": "S'enregistrer",
-    "not_logged_in": "Non connecté !",
+    "not_logged_in": "Non connecté",
     "something_went_wrong": "Quelque chose s'est mal passé !",
     "loading": "Chargement",
+    "common": {},
     "account": {
         "home": {
             "title": "Compte",
             "update_password": "Mettre à jour votre mot de passe",
-            "first_time_message": "Si c'est la première fois que vous vous connectez sur ce compte utilisateur, vous devez accéder aux paramètres du compte et mettre à jour votre mot de passe",
+            "first_time_message": "Si c'est la première fois que vous vous connectez sur ce compte utilisateur, vous devriez éditer les paramètres du compte et modifier votre mot de passe",
```

Comment:
@bjarneo I'd like your confirmation here: in English, it is "you should go to Account settings and update your password" which does not sound like an obligation. In French, it was "vous devez" which is the direct translation of "you must". I changed it to softer "you should" but if this password change is mandatory, I _should_ revert my change because the previous better was better in this case.

Reply:
You are correct here. The English text should be updated to be clear. "You must" is correct here.

Reply:
Got it, thanks for confirming. I'll revert this change and update the English version accordingly, then.

Reply:
Done.